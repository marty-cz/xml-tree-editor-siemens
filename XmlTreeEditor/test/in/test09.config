<config>
  <input>test09.xml</input>
  <schema>schema01.xsd</schema>
</config>