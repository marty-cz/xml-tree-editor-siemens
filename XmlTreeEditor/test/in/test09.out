Error: Schema file "test01.xsd" does not correspond with input xml file "test02.xml".